airbyte-config/models/src/main/java/io/airbyte/config/Configs.java

```diff
@@ -31,6 +31,8 @@
 
   String getAirbyteRole();
 
+  String getAirbyteServerHost();
```

Comment:
Why this change?

Reply:
I need it in the cloud to know where the airbyte server is deployed to call the API